src/app/dataset-view/additional-components/metadata-component/components/set-transform/components/queries-section/queries-section.component.ts

```diff
@@ -0,0 +1,78 @@
+import {
+    ChangeDetectionStrategy,
+    Component,
+    Input,
+    OnChanges,
+    SimpleChanges,
+} from "@angular/core";
+import { SqlQueryStep } from "src/app/api/kamu.graphql.interface";
+import * as monaco from "monaco-editor";
+import { ViewportScroller } from "@angular/common";
+
+@Component({
+    selector: "app-queries-section",
+    templateUrl: "./queries-section.component.html",
+    styleUrls: ["./queries-section.component.sass"],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class QueriesSectionComponent implements OnChanges {
+    @Input() public queries: Omit<SqlQueryStep, "__typename">[];
+    @Input() public datasetName: string;
+    public readonly sqlEditorOptions: monaco.editor.IStandaloneEditorConstructionOptions =
+        {
+            theme: "vs",
+            language: "sql",
+            renderLineHighlight: "none",
+            minimap: {
+                enabled: false,
+            },
+        };
```

Comment:
There exists another definition already in `DatasetCreateComponent`, and I see no reason why we need two. Place it somewhere in common directory.

Reply:
Move to config file